jobs/ci-run/functional/builders.yaml

```diff
@@ -0,0 +1,19 @@
+# Common builders needed for functional tests to continue running
+
+- builder:
+    name: 'functional-python-deps'
+    builders:
+    - shell: |-
+        #!/bin/bash
+        set -eux
+
+        pip_deps="pywinrm boto pytz"
+        if ! [[ "{modules}" =~ ^.modules.$ ]]; then
+          pip_deps="$pip_deps {modules}"
+        fi
+
+        if which pip; then
```

Comment:
command -v
instead of which? i think that's what's preferred since it's built in